server/TypeDBServer.java

```diff
@@ -71,24 +71,28 @@ public class TypeDBServer implements AutoCloseable {
 
     protected final Factory factory;
     protected final CoreConfig config;
-    protected final CoreLogback logback;
```

Comment:
this is an uneeded field, it is just required to configure the logger which we can do statically